extensions/interactions/ItemSelectionInput/protractor.js

```diff
@@ -55,7 +55,7 @@ var customizeInteraction = async function(
 var expectInteractionDetailsToMatch = async function(
     elem, richTextInstructionsArray) {
   var optionElements = elem.all(
-    by.repeater('choice in $ctrl.choices track by $index'));
+    by.css('.item-selection-input-container'));
```

Comment:
If you want to change it, please follow [this ](https://github.com/oppia/oppia/wiki/End-to-End-Tests#selecting-elements)guideline as mentioned on the wiki page.

> Adding a protractor-test-some-name class to the element in question, and then referencing it by by.css('.protractor-test-some-name').

Also, can you can please explain the reason behind the change.

Reply:
Done,

after migration of its related file, syntax for loop is change.